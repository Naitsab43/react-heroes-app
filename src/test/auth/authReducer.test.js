import React from "react"
const { shallow } = require("enzyme")
const { authReducer } = require("../../auth/authReducer")
import "@testing-library/jest-dom"

describe('Pruebas en authReducer', () => {
  
  test('Debe de retornar el estado por defecto', () => {
    
    const wrapper = shallow(<authReducer />)

    expect(wrapper).toMatchSnapshot()

  })
  

  test('Debe de autenticar y colocar el name del usuario', () => {
    
  })
  

})
