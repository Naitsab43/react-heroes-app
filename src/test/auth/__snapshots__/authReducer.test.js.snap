// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en authReducer Debe de retornar el estado por defecto 1`] = `<authReducer />`;
